http4k-contract-ui/redoc/src/main/kotlin/org/http4k/contract/ui/redoc/redocWebjar.kt

```diff
@@ -0,0 +1,26 @@
+package org.http4k.contract.ui.redoc
+
+import org.http4k.contract.ui.RedocConfig
+import org.http4k.contract.ui.toFilter
+import org.http4k.core.Method.GET
+import org.http4k.core.Response
+import org.http4k.core.Status.Companion.FOUND
+import org.http4k.core.appendToPath
+import org.http4k.core.then
+import org.http4k.core.with
+import org.http4k.lens.Header
+import org.http4k.routing.ResourceLoader
+import org.http4k.routing.bind
+import org.http4k.routing.routes
+import org.http4k.routing.static
+
+fun redocWebjar(configFn: RedocConfig.() -> Unit = {}) = routes(
+    "" bind GET to { Response(FOUND).with(Header.LOCATION of it.uri.appendToPath("index.html")) },
```

Comment:
Even though `static` will implicitly load `index.html` for a bare path, this causes issues when the HTML is attempting to load relative scripts.  Same applied to Swagger UI.

Reply:
Nice fix 🙃